GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/StorageClient.DownloadObject.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Download;
+using Google.Common;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Object = Google.Apis.Storage.v1.Data.Object;
```

Comment:
It's tricky. I think in the context, we're okay with just Object - it might look a little odd if someone were to compare the generated documentation (which will use Object) and the source code. When I need to use System.Object, I'd use "object" anyway.

Definitely up for discussion, but fortunately API-neutral anyway.